app/views/courses/TeacherClassesView.coffee

```diff
@@ -71,6 +129,45 @@ module.exports = class TeacherClassesView extends RootView
 
   onLoaded: ->
     helper.calculateDots(@classrooms, @courses, @courseInstances)
+
+    @teacherQuestData['create_classroom'].complete = @classrooms.length > 0
+    for classroom in @classrooms.models
+      continue unless classroom.get('members')?.length > 0
+      classCompletion = {}
+      classCompletion[key] = 0 for key in Object.keys(@teacherQuestData)
+      students = classroom.get('members')?.length
+
+      
+      kithgardGatesCompletes = 0
+      wakkaMaulCompletes = 0
+      for session in classroom.sessions.models
+        if session.get('level')?.original is '541c9a30c6362edfb0f34479' # kithgard-gates
+          ++classCompletion['kithgard_gates_100']
+        if session.get('level')?.original is '5630eab0c0fcbd86057cc2f8' # wakka-maul
+          ++classCompletion['wakka_maul_100']            
+        continue unless session.get('state')?.complete
+        if session.get('level')?.original is '5411cb3769152f1707be029c' # dungeons-of-kithgard
+          ++classCompletion['teach_methods']
+        if session.get('level')?.original is '541875da4c16460000ab990f' # true-names
+          ++classCompletion['teach_strings']
+        if session.get('level')?.original is '55ca293b9bc1892c835b0136' # fire-dancing
+          ++classCompletion['teach_loops']
+        if session.get('level')?.original is '5452adea57e83800009730ee' # known-enemy
+          ++classCompletion['teach_variables']
+
+      classCompletion[k] /= students for k of classCompletion
+        
+
+
+      classCompletion['add_students'] = if students > 0 then 1.0 else 0.0
+      if me.get('enrollmentRequestSent') or @prepaids.length > 0
+        classCompletion['reach_gamedev'] = 1.0
+      else
+        classCompletion['reach_gamedev'] = 0.0
+
+      @teacherQuestData[k].complete ||= v > 0.74 for k,v of classCompletion
```

Comment:
Is this not `>= 0.75` because of rounding logic? Would prefer that be done explicitly if so